src/portable/raspberrypi/rp2040/dcd_rp2040.c

```diff
@@ -67,11 +65,7 @@ static struct hw_endpoint *hw_endpoint_get_by_addr(uint8_t ep_addr)
 }
 static void _hw_endpoint_alloc(struct hw_endpoint *ep)
 {
-    uint size = 64;
-    if (ep->wMaxPacketSize > 64)
-    {
-        size = ep->wMaxPacketSize;
-    }
+    uint size = TU_MIN(64, ep->wMaxPacketSize);
```

Comment:
larger than 64 actually should throw the error, but it won't get this far. USBD will rise an TU_ASSERT()
https://github.com/hathach/tinyusb/blob/master/src/device/usbd.c#L1112